app/controllers/photos_controller.rb

```diff
@@ -8,6 +8,7 @@ def find_album
 
   def find_photo
     @photo = Photo.find(params[:id])
+    @photo.view += 1;
```

Comment:
yes, I just realised that when I checked my notes. I should have used the increment! method, not the counter =+. 
Thank you